runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -7794,6 +7809,23 @@ class INTERPRETER_CLASS
 		/* Swap flags and class subfield order. */
 		classAndFlags = J9CLASSANDFLAGS_FROM_FLAGSANDCLASS(flagsAndClass);
 		valueAddress = J9STATICADDRESS(flagsAndClass, valueOffset);
+
```

Comment:
do we need ifdefs here as well

Reply:
Its highly likely we will incur somekind of interpreter regression with these changes. I'd like to limit it to JDK levels that actually make use of valuetypes

Reply:
I'll add a new build flag.